docs/use-github-in-bcgov/start-working-in-bcgov-github-organization.md

```diff
@@ -62,6 +62,15 @@ In both cases, the basic steps to release the code are similar, while the implic
 
 If you are intending to maintain an active project, make sure to establish the appropriate processes and terms to manage contributions.
 
+## Integrating GitHub Apps
+
+We have large volumes of asks about adding GitHub Apps to our organization’s repository. To manage the asks, we created a request process through the devops-requests repository.
```

Comment:
please link to the devops request repository here.

Reply:
Here is the link to the specific DevOps request for requesting a GitHub App. I've reworked the sentence to use the link. Please feel free to revert back to previous sentence.


```suggestion
We have large volumes of asks about adding GitHub Apps to our organization’s repository. Request a GitHub App to be installed from the BCDevOps Request Issue: [Request for integrating a GitHub App](https://github.com/BCDevOps/devops-requests/issues/new?assignees=MonicaG%2C+oomIRL%2C+SHIHO-I&labels=github-app%2C+pending&projects=&template=github_integration_request.md&title=).
```